drainer/syncer.go

```diff
@@ -562,6 +654,37 @@ func filterTable(pv *pb.PrewriteValue, filter *filter.Filter, schema *Schema) (i
 	return
 }
 
+// skipDDLEvent may drop some ddl event
+// Return true if this job is filtered.
+func skipDDLEvent(sql, schema, table string, p *parser.Parser, binlogFilter *bf.BinlogEvent) (ignore bool, err error) {
+	if binlogFilter == nil {
+		return false, nil
+	}
+	stmt, err := p.ParseOneStmt(sql, "", "")
+	if err != nil {
+		log.L().Error("fail to parse ddl", zap.String("ddl", sql), logutil.ShortError(err))
+		// return error if parse fail and filter fail
+		needSkip, err2 := skipByFilter(binlogFilter, schema, table, bf.NullEvent, sql)
+		return needSkip, err2
```

Comment:
IF needSkip is false here, do we need to return a error as this is an unknown statement.

Reply:
I think that doesn't matter. If this can't be executed downstream they will return an error.